### PR TITLE
1.2.0a2: `installer.remove_untracked` -> adds installer.requires_synchronization(True)

### PR DESCRIPTION
**Pull request by jtzero**
the function has been moved to Solver

fixes error 
```

  AttributeError

  'Installer' object has no attribute 'remove_untracked'

  at ~/.asdf/installs/python/3.8.5/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/poetry_bundle_plugin/bundlers/venv_bundler.py:133 in bundle
      129│             poetry.locker,
      130│             poetry.pool,
      131│             poetry.config,
      132│         )
    → 133│         installer.remove_untracked()
      134│         installer.use_executor(poetry.config.get("experimental.new-installer", False))
      135│
      136│         return_code = installer.run()
      137│         if return_code:
```

### CODE REVIEW

**Comment by MaciejKucia** [comment]
@sdispater Can this fix be merged? Thanks.
As an alternative what would be the compatible poetry version?

**Comment by talhajunaidd** [comment]
I'm also facing the same issue, can we please merge the fix?

**Comment by jwillikers** [comment]
Same issue.

**Comment by talhajunaidd** [comment]
any update?

**Comment by mgb-ingenuity** [comment]
What's the status on this?

**Comment by RockyMM** [comment]
Just to add -> same issue. Any chance for a merge?

**Comment by mgb-ingenuity** [comment]
BUMP

File: src/poetry_bundle_plugin/bundlers/venv_bundler.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def bundle(self, poetry: "Poetry", io: "IO") -> bool:
             poetry.pool,
             poetry.config,
         )
-        installer.remove_untracked()
+        installer.requires_synchronization(True)
```
Comment (neersighted) [inline src/poetry_bundle_plugin/bundlers/venv_bundler.py:133]: There should be no need to explicitly pass the default argument here. Otherwise, LGTM.

**Review by neersighted — CHANGES_REQUESTED** [review]

**Review by branchvincent — APPROVED** [review]
thanks @jtzero, and sorry for the long delay. I'll try to get this published asap

**Pull request merged**